include/openssl/ecerr.h

```diff
@@ -173,8 +173,6 @@ int ERR_load_EC_strings(void);
 #  define EC_F_PKEY_ECD_DIGESTSIGN                         272
 #  define EC_F_PKEY_ECD_DIGESTSIGN25519                    276
 #  define EC_F_PKEY_ECD_DIGESTSIGN448                      277
-#  define EC_F_PKEY_ECD_SIGN25519                          284
-#  define EC_F_PKEY_ECD_SIGN448                            285
```

Comment:
This is not right way to handle public header in minor release.

Reply:
These were newly introduced and haven't been in a stable release yet.